### ISSUE TITLE
add bootstrap validator for sign up forms

### ISSUE THREAD

**Issue by kas70** [diaspora/diaspora#5827]
example screenshot:  
![validator](https://cloud.githubusercontent.com/assets/922628/6916979/d188c55e-d7a0-11e4-8c96-900234fdd349.jpg)  
maybe also for already taken usernames.  
example code: https://1000hz.github.io/bootstrap-validator/

**Comment by efstrian** [comment]
Could this really be implemented? One should change the looks of the sign up forms (spacing between fields, multiple error messages), shouldn't he?

**Comment by svbergerem** [comment]
I agree that this would need some additional changes on the sign up page. Let's decide that in the (hopefully coming) redesign.
